### ISSUE TITLE
Can it be launched with Firegestures addon ?

### ISSUE THREAD

**Issue by LL25255252** [DanielKamkha/PlayPauseFirefox#30]
Play-pause addon seems to be a great addon and I want to use it but as of now I can only use it with its keyboard shortcut.

The thing is that I use "Tab Utilities Fixed" (TUF) addon which enables me to click the current tab to get its history-menu.
https://addons.mozilla.org/en-US/firefox/addon/tab-utilities-fixed/
When installing both P/P addon anf TUF there's a problem. Clicking on the play/pause icon of a not-current-tab it launches the play/pause but it switches the focus to that tab.

So I need another "method" to play/pause those not-current-tab.

Another important thing (this time it doesn't matter if TUF is  installed or not) : **clicking on a very small icon is not comfortable** and I guess many users will prefer other methods.

One option is to click the P/P keyboard shortcust (this option works ok for me but I want a better option - I always prefer not to use keyboard shortcuts).

**So I have some suggestions:**

1. Can you add a play/pause menu-item to to the tab context menu or the main context-menu ?

2. a play/pause toolbar button

3. Can you post the code/command which is needed to create a firegesture script to launch the play/pause.
https://addons.mozilla.org/en-Us/firefox/addon/firegestures/

Here's an example with another addon: 
I installed the "focus Last Tab" addon.
https://addons.mozilla.org/en-US/firefox/addon/focus-last-tab/?src=api
Installing https://addons.mozilla.org/en-US/firefox/addon/dorando-keyconfig/  lets you see that  "focus Last Tab" addon is associated with this command :  focuslasttab.swap()
Then I created a new gesture and associated it with focuslasttab.swap()

4. If you give me the code/command (as in suggestion [3]) I can try to create a new tab-context-menu with the Menu Wizard addon.
https://addons.mozilla.org/en-US/firefox/addon/s3menu-wizard/

Cyberfox 51.0.3 (32-bit)
Win 7 x64

Thanks in advance.

**Comment by DanielKamkha** [comment]
OK, that's a lot of stuff.

1. TUF compatibility issue is definitely something I'll be looking into, albeit with lower priority than the bugs in Play/Pause itself.
2. Toolbar button is already in the works. I want to make it not to appear by default, and there are few more other minor things unfinished for the button.
3. Adding a line in the context menu would be awesome. However, to my knowledge, such functionality makes an addon to require restart on install/update. Which is a big no-no in my book.
4. Making sense of figestures and providing something for it is in the same bucket as TUF compatibility. But as it is not even a bug, the priority is even lower.

tl;dr:
TUF compatibility fix: yes
Toolbar button: soon(tm)
The rest of stuff: no

**Comment by DanielKamkha** [comment]
Tab Utilities Fixed is marked as "experimental". I will revisit this compatibility issue when TUF becomes stable.

**Comment by LL25255252** [comment]
...yes you're right. It seems that the problem is with TUF. It has a similar conflict with the browser's built-in mute button.
I just began following this open issue. Here - https://github.com/yfdyh000/tabutils/issues/134

**Comment by LL25255252** [comment]
@DanielKamkha , is there a chance that you can help with a fix to these addons conflict ?
See TUF's dev last comment - https://github.com/yfdyh000/tabutils/issues/134#issuecomment-284281002

**Comment by DanielKamkha** [comment]
Unlikely. As far as I understand Firefox release schedule, all addons that are not WebExtensions (this includes TUF) will sorta "die" somewhere around this November. Unless they convert to WebExtensions, which, I suspect, is not even possible for TUF.

So please enjoy TUF while you can, and I will spend my time transitioning my own addons to WebExtensions instead.
